Auto_File_Organization.py

```diff
@@ -1,83 +1,114 @@
+    },
+    'Audio Files': {
+        'Extensions': ['aif', 'cda', 'iff', 'mid', 'midi', 'mp3', 'mpa', 'wav', 'wma', '3g2', '3gp'],
+        'File Names': ['Audio Interchange File Format (aif)', 'Compact Disc Audio Track (cda)', 'Interchange File Format (iff)', 'MIDI File (mid)', 'MIDI File (midi)', 'MP3 Audio File (mp3)', 'MPEG Audio (mpa)', 'Waveform Audio File Format (wav)', 'Windows Media Audio (wma)', '3GPP2 Multimedia File (3g2)', '3GPP Multimedia File (3gp)']
+    },
+    'Video Files': { 
+        'Extensions': ['mp4', 'avi', 'flv', 'mkv', 'mov', 'wmv'],
+        'File Names': ['MPEG-4 Video File (mp4)', 'Audio Video Interleave (avi)', 'Flash Video (flv)', 'Matroska Video File (mkv)', 'Apple QuickTime Movie (mov)', 'Windows Media Video (wmv)']
+    },
+    'Spreadsheets': {
+        'Extensions': ['ods', 'xlr', 'xls', 'xlsx'],
+        'File Names': ['OpenDocument Spreadsheet (ods)', 'Microsoft Works Spreadsheet (xlr)', 'Microsoft Excel Spreadsheet (xls)', 'Microsoft Excel Spreadsheet (xlsx)']
+    },
+    'Presentations': {
+        'Extensions': ['key', 'odp', 'pps', 'ppt', 'pptx'],
+        'File Names': ['Keynote Presentation (key)', 'OpenDocument Presentation (odp)', 'PowerPoint Slide Show (pps)', 'PowerPoint Presentation (ppt)', 'PowerPoint Presentation (pptx)']
+    },
+    'Database Files': {
+        'Extensions': ['accdb', 'csv', 'dat', 'db', 'dbf', 'log', 'mdb', 'pdb', 'sav', 'sql', 'tar'],
+        'File Names': ['Access Database (accdb)', 'Comma-Separated Values (csv)', 'Data File (dat)', 'Database File (db)', 'Database File (dbf)', 'Log File (log)', 'Microsoft Database (mdb)', 'Palm Desktop Database (pdb)', 'SPSS Data File (sav)', 'Structured Query Language (sql)', 'Tape Archive (tar)']
+    },
+    'Web Files': {
+        'Extensions': ['asp', 'aspx', 'cer', 'cfm', 'cgi', 'pl', 'css', 'htm', 'html', 'js', 'part', 'php', 'rss', 'xhtml'],
+        'File Names': ['Active Server Pages (asp)', 'Active Server Page Executable (aspx)', 'Security Certificate (cer)', 'ColdFusion Markup Language File (cfm)', 'Common Gateway Interface Script (cgi)', 'Perl Script (pl)', 'Cascading Style Sheet (css)', 'Hypertext Markup Language (htm)', 'Hypertext Markup Language (html)', 'JavaScript File (js)', 'Partially Downloaded File (part)', 'Hypertext Preprocessor (php)', 'Really Simple Syndication (rss)', 'Extensible Hypertext Markup Language (xhtml)']
+    },
+    'System Files': {
+        'Extensions': ['bak', 'cab', 'cfg', 'cpl', 'cur', 'dll', 'dmp', 'drv', 'icns', 'ico', 'ini', 'lnk', 'msi', 'sys', 'tmp'],
+        'File Names': ['Backup File (bak)', 'Windows Cabinet File (cab)', 'Configuration File (cfg)', 'Control Panel Extension (cpl)', 'Cursor File (cur)', 'Dynamic Link Library (dll)', 'Dump File (dmp)', 'Device Driver (drv)', 'Icon File (icns)', 'Icon File (ico)', 'Configuration File (ini)', 'Shortcut (lnk)', 'Windows Installer Package (msi)', 'System File (sys)', 'Temporary File (tmp)']
+    }
 }

```

Comment:
Hi @daharupesh,

To improve the separation of concerns and enhance code readability and maintainability, we can create a separate `fileTypes.json` file. This will allow us to store file type information separately from the code logic. You can use `Pandas` to read the JSON file, ensuring that the data is handled efficiently and cleanly.

Reply:
Hi @sanjaysah101 ,

Thank you for the suggestion. That’s a great idea! I'll create a `fileTypes.json` file to separate the file type information from the code logic. Using Pandas to read the JSON file will certainly improve the maintainability and readability of the code. I'll implement these changes and push the updates soon.